controller.go

```diff
@@ -239,6 +242,12 @@ func addAlarmDefinition(r Resource) error {
 	if !strings.HasSuffix(r.Spec.Name, alarmDefinitionControllerSuffix) {
 		r.Spec.Name = r.Spec.Name + alarmDefinitionControllerSuffix
 	}
+	if *defaultNotification != "" && len(r.Spec.AlarmActions) <= 0 {
+		if defaultNotificationID == "" {
+			return errors.New("Unable to apply default notification method: no ID found")
```

Comment:
do we need to error this everytime? also it would never get here as you poll forever in the other loop if notification method is set

Reply:
That other loop is a goroutine

Reply:
duh